source/_components/rflink.markdown

```diff
@@ -99,9 +116,9 @@ sensor:
     automatic_add: true
 ```
 
-[RFLink Switches](https://www.home-assistant.io/components/switch.rflink/) cannot be added automatically. 
+[RFLink Switches](https://www.home-assistant.io/components/switch.rflink/) and [RFLink Binary Sensors](https://www.home-assistant.io/components/binary_sensor.rflink/) cannot be added automatically. 
 
-The RFLink component does not know the difference between a switch and a light. Therefore all switchable devices are automatically added as light by default. However, once the ID of a switch is known, it can be used to configure it as a switch type in HA, for example, to add it to a different group, hide it or configure a nice name.
+The RFLink component does not know the difference between a binary sensor, aswitch and a light. Therefore all switchable devices are automatically added as light by default. However, once the ID of a switch is known, it can be used to configure it as a switch or a binary sensor type in Home Assistant, for example, to add it to a different group, hide it or configure a nice name.
```

Comment:
```suggestion
The RFLink component does not know the difference between a binary sensor, a switch and a light. Therefore all switchable devices are automatically added as light by default. However, once the ID of a switch is known, it can be used to configure it as a switch or a binary sensor type in Home Assistant, for example, to add it to a different group, hide it or configure a nice name.
```